src/Tizen.NUI/src/public/Common/PropertyMap.cs

```diff
@@ -41,13 +44,43 @@ public PropertyMap() : this(Interop.PropertyMap.NewPropertyMap(), true)
         /// <since_tizen> 3 </since_tizen>
         public PropertyMap(PropertyMap other) : this(Interop.PropertyMap.NewPropertyMap(PropertyMap.getCPtr(other)), true)
         {
+            if (other != null)
+            {
+                propertyMap = new Dictionary<PropertyKey, PropertyValue>(other.propertyMap);
+            }
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
         internal PropertyMap(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
```

Comment:
`propertyMap` not created when use this constructor `internal PropertyMap(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)`

Reply:
Done.
propertyMap has been removed.